k8s/auto-inst/Dockerfile

```diff
@@ -7,4 +7,6 @@ COPY . .
 
 RUN npm install ./dd-trace.tgz
 
+FROM busybox
+
```

Comment:
We need to copy the library from the build image to the new one `COPY --from=build <src> <dst>`

Reply:
oh gotcha. will do.